src/utils/Api.js

```diff
@@ -0,0 +1,114 @@
+class Api {
+  constructor({baseUrl, headers}){
+    this._baseUrl = baseUrl;
+    this._headers = headers;
+  }
+
+
+  getAppInfo() {
+  return Promise.all([this.getInitialCards(), this.getUserInfo()]);
+  }
+
+  _checkResponse(res){
+    if(res.ok){
+      console.log(res)
+      return res.json();
+    }
+    return Promise.reject(`Error: ${res.status}`);
+  }
+
+  getInitialCards() {
+    return fetch(`${this._baseUrl}/cards`, {
+      headers: this._headers,
+    }).then(this._checkResponse);
+  }
+
+
+
+  getUserInfo() {
+    return fetch(`${this._baseUrl}/users/me`, {
+      headers: this._headers,
+    }).then((res) => {
+      if(res.ok){
+        return res.json();
+      }
+      Promise.reject(`Error: ${res.status}`);
+    });
```

Comment:
Please, use `.then(this._checkResponse);` in each request to avoid the code duplication